prep_bm_host.sh

```diff
@@ -342,7 +342,7 @@ if [[ "$PREP_FLAG" != "--skip-ocp-binaries" ]]; then
 
         curl -O "$OCP_CLIENT_BINARY_URL"
         tar xvf "${OCP_CLIENT_BINARY_URL##*/}"
-        sudo mv oc "$REQUIREMENTS_DIR"
+        sudo mv oc kubectl "$REQUIREMENTS_DIR"
```

Comment:
Does `kubectl` exist in the `/tmp` directory?  I'm not sure our script actually downloads it.

Reply:
Yes, it's contained in the same tarball of openshift-client-linux.